### PR TITLE
Added new article to the Layer5 Newsroom.

### LINKED ISSUE #2822: [News] Intel, Layer 5 Announce MeshMark to Quantify Cloud-Native Performance
We're in the news! Using the contributing instructions for [adding news](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-news), please add the following article to the [Layer5 Newsroom](https://layer5.io/news).

**Article:**
- Title: Intel, Layer 5 Announce MeshMark to Quantify Cloud-Native Performance
- Link: https://www.itprotoday.com/performance-management/intel-layer-5-announce-meshmark-quantify-cloud-native-performance

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by vishal-codes**
Signed-off-by: vishal-codes <itsvishal2417@gmail.com>

**Description**
Added the news article titled - "Intel, Layer 5 Announce MeshMark to Quantify Cloud-Native Performance"  to Layer5 Newsroom 

This PR fixes #2822 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f4fa8f9925147385db3a0e94652625a8782d0a02 at: https://6288ec9734ec3d11cc159989--layer5ng.netlify.app

**Review by Akshit42-hue — APPROVED** [review]
LGTM! Great work @vishal-codes

**Comment by vishal-codes** [comment]
> LGTM! Great work @vishal-codes

thank you @Akshit42-hue 🥳

File: src/collections/news/2022/2022-05-21-intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance/intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+MeshMark, an open source effort jointly announced by Intel and service mesh startup Layer5 on May 18, looks to help organizations measure and quantify the performance and value of a service mesh deployment. MeshMark is part of the Service Mesh Performance project at the Cloud Native Computing Foundation (CNCF), which hosted its ServiceMeshCon EU event on May 18, co-located alongside KubeCon EU 2022.
+
+"We're missing some performance characteristics," Lee Calcote, founder and CEO of Layer5 and co-chair of the CNCF Technical Advisory Group (TAG) Network, said during a ServiceMeshCon EU session. "We have a need for a clear and concise way of conveying the characters and the performance of an environment."
+
+MeshMark Brings Metrics to Cloud-Native Service Mesh. The Service Mesh Performance project at its core is an effort to define specifications for capturing the details of a cloud-native environment in a uniform and consistent way, according to Calcote.
```
Comment (Chadha93) [inline src/collections/news/2022/2022-05-21-intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance/intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance.md:35]: This line is repeated in line 28, please remove this.

File: src/collections/news/2022/2022-05-21-intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance/intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+The open source Service Mesh Performance project is getting a new metric called MeshMark to help organizations manage and measure cloud-native environments.
+
+In cloud-native environments, the use of service mesh technologies to connect different operations is a growing trend. What isn't always clear with a service mesh though is how well it's actually working.
+
+MeshMark, an open source effort jointly announced by Intel and service mesh startup Layer5 on May 18, looks to help organizations measure and quantify the performance and value of a service mesh deployment. MeshMark is part of the Service Mesh Performance project at the Cloud Native Computing Foundation (CNCF), which hosted its ServiceMeshCon EU event on May 18, co-located alongside KubeCon EU 2022.
+
+"We're missing some performance characteristics," Lee Calcote, founder and CEO of Layer5 and co-chair of the CNCF Technical Advisory Group (TAG) Network, said during a ServiceMeshCon EU session. "We have a need for a clear and concise way of conveying the characters and the performance of an environment."
+
+MeshMark Brings Metrics to Cloud-Native Service Mesh
```
Comment (Chadha93) [inline src/collections/news/2022/2022-05-21-intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance/intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance.md:28]: Please use a line break, make this as a subheading for the below paragraph, or add punctuation.

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@vishal-codes please revert other file changes :)

File: src/collections/news/2022/2022-05-21-intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance/intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance.md (hunk 1 above)
Comment (vishal-codes) [reply src/collections/news/2022/2022-05-21-intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance/intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance.md:35]: ahh..ohh sorry! I'll fix it asap

File: src/collections/news/2022/2022-05-21-intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance/intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance.md (hunk 2 above)
Comment (vishal-codes) [reply src/collections/news/2022/2022-05-21-intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance/intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance.md:28]: On it!

**Comment by l5io** [comment]
🚀 Preview for commit a048602846d454feea0c6b87182282aa26800b2e at: https://628a256cef672b4b7acab307--layer5ng.netlify.app

**Comment by vishal-codes** [comment]
> @vishal-codes please revert other file changes :)

I am unable to revert them, I have no idea why this is happening, the changes to those test files were made automatically, i don't know how and if I try to undo them - it's of no avail. What should I do?

**Comment by Akshit42-hue** [comment]
Hey @vishal-codes, you can refer this article https://stackoverflow.com/questions/927358/how-do-i-undo-the-most-recent-local-commits-in-git

**Comment by leecalcote** [comment]
@vishal-codes, these have got to go: `index.test.js`. I know that you've tried hard to contribute here and I very much appreciate this. It looks like we're setup for failure here, again, though. I think.... that you'll need to examine your IDE, it's plugins, and disable the auto-generation of `index.test.js`. 

Given our unsuccessful history here, I'm going to close this PR and get the news posted.

**Comment by vishal-codes** [comment]
> @vishal-codes, these have got to go: `index.test.js`. I know that you've tried hard to contribute here and I very much appreciate this. It looks like we're setup for failure here, again, though. I think.... that you'll need to examine your IDE, it's plugins, and disable the auto-generation of `index.test.js`. 
> 
> Given our unsuccessful history here, I'm going to close this PR and get the news posted.

Oh okaay.. I'll examine the IDE. 👍

**Pull request closed without merging**